src/Maker/MakeAuthenticator.php

```diff
@@ -55,88 +71,275 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
+        } else {
+            $controllerPath = $this->fileManager->getRelativePathForFutureClass($controllerClassNameDetails->getFullName());
+            $controllerSourceCode = $this->fileManager->getFileContents($controllerPath);
+        }
+
+        if (method_exists($controllerClassNameDetails->getFullName(), 'login')) {
+            throw new RuntimeCommandException(sprintf('Method "login" already exists on class %s', $controllerClassNameDetails->getFullName()));
+        }
+
+        $manipulator = new ClassSourceManipulator($controllerSourceCode, true);
+
+        $securityControllerBuilder = new SecurityControllerBuilder();
+        $securityControllerBuilder->addLoginMethod($manipulator);
+
+        $this->generator->dumpFile($controllerPath, $manipulator->getSourceCode());
+
+        // create login form template
+        $this->generator->generateFile(
+            'templates/security/login.html.twig',
+            'authenticator/login_form.tpl.php',
+            [
+                'username_field' => $userNameField,
+            ]
+        );
+    }
+
+    private function generateNextMessage(bool $securityYamlUpdated, string $authenticatorType, string $authenticatorClass, array $securityData, $userClass): array
+    {
+        $nextTexts = ['Next:'];
+        $nextTexts[] = '- Customize your new authenticator.';
+
         if (!$securityYamlUpdated) {
             $yamlExample = $this->configUpdater->updateForAuthenticator(
                 'security: {}',
                 'main',
                 null,
-                $classNameDetails->getFullName()
+                $authenticatorClass
             );
-            $text[] = "Your <info>security.yaml</info> could not be updated automatically. You'll need to add the following config manually:\n\n".$yamlExample;
+            $nextTexts[] = '- Your <info>security.yaml</info> could not be updated automatically. You\'ll need to add the following config manually:\n\n'.$yamlExample;
+        }
+
+        if (self::AUTH_TYPE_FORM_LOGIN === $authenticatorType) {
+            $nextTexts[] = sprintf('- You must provide a valid redirection in the method <info>%s::onAuthenticationSuccess()</info>.', $authenticatorClass);
+            $nextTexts[] = '- Review & adapt the login template : <info>/templates/security/login.html.twig</info>.';
+
+            if (!$this->doctrineHelper->isClassAMappedEntity($userClass)) {
+                $nextTexts[] = sprintf('- Review <info>%s::getUser()</info>, if it match your needs.', $authenticatorClass);
+            }
+
+            if (!$this->userNeedsEncoder($securityData, $userClass)) {
+                $nextTexts[] = sprintf('- Check user\'s password in <info>%s::checkCredentials()</info>.', $authenticatorClass);
+            }
         }
-        $io->text($text);
+
+        return $nextTexts;
+    }
+
+    private function userNeedsEncoder(array $securityData, string $userClass): bool
```

Comment:
Could we name this `userClassHasEncoder()`? Is that the correct meaning of this?